Code/FooBar.php

```diff
@@ -20,6 +20,10 @@ public function checker($number)
             $resultArray[1] = "Bar";
         }
 
+        if ($number % 7 == 0) {
```

Comment:
During this step You have to notice, that You have code duplication. You are doing the same thing for three times.